content/en/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -16,8 +16,23 @@ be used when running Kubernetes on Amazon Web Services.
 
 ### Node Name
 
+#### Default
+
 The AWS cloud provider uses the private DNS name of the AWS instance as the name of the Kubernetes Node object.
 
+#### Explicit
+
+<aside class="warning"> Switching from default node names to explicit node names can orphan volumes</aside>
```

Comment:
I would recommend making this language stronger. It not only orphans volumes, it can also cause the kubelet credentials to become invalid, etc. In general, the name of an existing node should not be changed.